pkg/azuredisk/azuredisk.go

```diff
@@ -142,7 +152,9 @@ func newDriverV1(nodeID string) *Driver {
 	if err != nil {
 		klog.Fatalf("%v", err)
 	}
-	driver.getDiskThrottlingCache = cache
+	driver.diskThrottlingCache = cache
```

Comment:
`getDiskThrottlingCache` is for handling `GetDisk` operation throttling, don't make other changes in this PR, focus on this  feature with  this PR.

Reply:
I don't intend to make any functional changes. I needed to expose this for test coverage and ran in to this name.
The go guidelines don't recommend getters to have get* naming, let alone for fields.
https://golang.org/doc/effective_go#Getters

I can revert it, if you have any concerns?

Reply:
that's for `Get`, not `get`, maybe `diskGetThrottlingCache` is more appropriate, anyway, this is out of this PR scope

Reply:
Will revert. Thanks!